### PR TITLE
MF-388 Initial Helm chart

### PR DESCRIPTION
**Pull request by isavcic**
### What does this do?

Provides the initial Helm chart for the Mainflux IoT Platform

### Which issue(s) does this PR fix/relate to?
MF-388

### List any changes that modify/break current functionality

None, this is a separate subproject

### Have you included tests for your changes?

No

### Did you document any new/modified functionality?

Not yet, TBD

### Notes

None

### CODE REVIEW

**Review by drasko — APPROVED** [review]
LGTM

File: k8s/charts/mainflux/templates/ingress.yaml [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+      secretName: {{ .Values.ingress.tls.secret }}
+{{- end }}
+---
+apiVersion: v1
+kind: ConfigMap
+data:
+  # MQTT adapter:
+  "1883": "{{ .Release.Namespace }}/{{ .Release.Name }}-envoy:1883:PROXY"
+  "8883": "{{ .Release.Namespace }}/{{ .Release.Name }}-nginx-internal:8883:PROXY"
+#   # Vault:
```
Comment (drasko) [inline k8s/charts/mainflux/templates/ingress.yaml:112]: Do we need this dead code?

File: k8s/charts/mainflux/values.yaml [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  image:
+    pullPolicy: &default_image_pullPolicy "IfNotPresent"
+    repository: &default_image_repository "mainflux"
+    tag: &default_image_tag "0.9.1"
+
+ingress:
+  enabled: true
+  annotations: {}
+  labels: {}
+  # hostname: ""
```
Comment (drasko) [inline k8s/charts/mainflux/values.yaml:12]: Here also

**Review by drasko — CHANGES_REQUESTED** [review]

**Comment by isavcic** [comment]
Let's merge this in the devops repo instead. Closing.

**Pull request closed without merging**